birch

```diff
@@ -183,6 +183,12 @@ cmd() {
             ln -sf "$chan" .c
             kill -28 0 
         ;;
+        
+        '/previous'*)
```

Comment:
'prev' would be better.

Reply:
This is probably the best way:

```sh
        '/next'*) 
            chan=${cl[z = z + 1 >= ${#cl[@]} ? 0 : z + 1]} 
        ;;&

        '/prev'*) 
            chan=${cl[z = z - 1 < 0 ? ${#cl[@]} : z - 1]}
        ;;&

        '/next'*|'/prev'*)
            ln -sf "$chan" .c
            kill -28 0 
        ;;
```

Reply:
It might be better like this. I will test it. Actually i had done all in on case with an internal case in case but it was horrible..

Reply:
Implmented it like this. Didn't know about ;;& . Thanks for the suggestion 👍

Reply:
`;&` will fall through regardless of checking the next pattern.

`;;&` will fall through and check the patterns.